coalib/bearlib/languages/definitions/Java.py

```diff
@@ -0,0 +1,12 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class Java:
+    extensions = '.java'
+    comment_delimiter = '//'
+    multiline_comment_delimiters = {'/*': '*/'}
+    string_delimiters = {'"': '"', "'": "'"}
```

Comment:
but we haven't had them listed yet and I'm not going to resarch now :P we can complete them later